### PR TITLE
Added import/export settings

### PR DESCRIPTION
**Pull request by Anready**
BEFORE COMPILING: PASS.FOR.ZIP=YOUR_PASS (VERY STRONG PASSWORD) in local.properties

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -4,6 +4,9 @@ plugins {
     id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
 }
 
+def Properties properties = new Properties()
```
Comment (coderGtm) [inline app/build.gradle:7]: use the current approach of the project for getting local.properties data, like the weatherAPIkey.

File: app/build.gradle [hunk 2]
```diff
@@ -15,6 +18,8 @@ android {
         versionName "9.6.2"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        resValue "string", "PASS_FOR_ZIP", properties.getProperty("PASS.FOR.ZIP", "")
```
Comment (coderGtm) [inline app/build.gradle:22]: use the current approach of the project for getting local.properties data, like the weatherAPIkey.

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -17,6 +17,7 @@
         android:label="${appLabel}"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        tools:replace="android:label"
```
Comment (coderGtm) [inline app/src/main/AndroidManifest.xml:20]: why added this?

File: settings.gradle [hunk 4]
```diff
@@ -11,6 +11,7 @@ dependencyResolutionManagement {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://jitpack.io' }
```
Comment (coderGtm) [inline settings.gradle:14]: why?

**Review by coderGtm — CHANGES_REQUESTED** [review]
Please read the single comments i put. Also, please don't add string resources. just add the strings and I will extract them after merging.

File: app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (Anready) [reply app/src/main/AndroidManifest.xml:20]: without this happens compile error

File: settings.gradle (hunk 4 above)
Comment (Anready) [reply settings.gradle:14]: we use external library MZip, without this library cannot be downloaded and registered

Comment (Anready) [reply settings.gradle:14]: we use external library MZip, without this library cannot be downloaded and registered

**Comment by Anready** [comment]
Okay np, so i need just change import strings from local properties, right?

**Comment by coderGtm** [comment]
Yeah

**Comment by coderGtm** [comment]
And also the strings...

**Comment by Anready** [comment]
yeah i remember

**Comment by Anready** [comment]
i hope i fixed everything

**Review by coderGtm — APPROVED** [review]

**Pull request merged**